operator/Makefile

```diff
@@ -112,11 +112,11 @@ RUN = $(CONTAINER_CLI) run -t -i --sig-proxy=true -u $(UID):$(GID) --rm \
 	-v /etc/passwd:/etc/passwd:ro \
 	$(DOCKER_SOCKET_MOUNT) \
 	$(CONTAINER_OPTIONS) \
-	--mount type=bind,source="$(PWD)",destination="/work" \
+	--mount type=bind,source="$(PWD)/../",destination="/work" \
```

Comment:
this file shouldn't be used and the licenses files under operator shouldn't be checked in (they are duplicates of too level). We should not be calling make within make, just including other makefiles

Reply:
Fixed